etc/kayobe/pulp.yml

```diff
@@ -10,6 +10,9 @@ pulp_url: "http://{{ admin_oc_net_name | net_ip(groups['seed'][0]) }}:80"
 pulp_username: admin
 pulp_password: "{{ secrets_pulp_password }}"
 
+# Proxy to use when adding remotes
+pulp_proxy_url: "{{ omit }}"
```

Comment:
In Yoga we get proper support for proxies in kayobe: https://review.opendev.org/c/openstack/kayobe/+/814046

Should this default to `https_proxy` when available? Either we could backport the proxy patch downstream, or add a comment here to update the default.

Reply:
Just to clarify, I set this in https://github.com/stackhpc/stackhpc-kayobe-config/pull/36/commits/5b24e5b4f27fd4991c1704c2aa0aff73b2965184 and then reverted it as we realised it was only a sensible default when using an external proxy. If the proxy is deployed on the seed, we probably don't want to use it.